src/tokens/universal.tokens.json

```diff
@@ -3097,6 +3097,30 @@
             }
           }
         },
+        "rewards-panel": {
+          "description": "",
+          "type": "custom-gradient",
+          "value": {
+            "gradientType": "linear",
+            "rotation": 134.9999995731132,
+            "stops": [
+              {
+                "position": 0,
+                "color": "#514de0ff"
+              },
+              {
+                "position": 1,
+                "color": "#af1768ff"
+              }
+            ]
+          },
+          "extensions": {
+            "org.lukasoppermann.figmaDesignTokens": {
+              "styleId": "S:7d40c65bebc9cad2488922f2c2b931edb969a481,",
+              "exportKey": "gradient"
+            }
+          }
+        },
```

Comment:
Just throwing this out there as a question: should `rewards-panel` be a part of the design system, or is that getting too project specific? If it's not meant to be used elsewhere, it seems the rewards panel code base should own this, and if it is meant to be used elsewhere, it seems it should be named to something more generic.

Reply:
:+1: to this - I think it'd be ideal if we could avoid adding project specific properties to Leo - if we have some legitimate additions, they can probably be added at a project specific level.

Reply:
cc @aguscruiz